pythran/pythonic/include/numpy/interp.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef PYTHONIC_INCLUDE_NUMPY_INTERP_HPP
```

Comment:
as this code is inspired from the numpy code base, you should state the original license and source used to derive this code.